### PR TITLE
sumo: update to 1.12.0.

### PR DESCRIPTION
**Pull request by tibequadorian**
Might help https://github.com/void-linux/void-packages/pull/34729

#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/sumo/template [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+makedepends="python3-devel ffmpeg-devel fox-devel gl2ps-devel libgdal-devel
+ libxerces-c-devel osg-devel proj-devel"
 short_desc="SUMO (Simulation of Urban MObility) and included applications"
 maintainer="Warrior Graph <marques@larces.uece.br>"
 license="EPL-2.0"
 homepage="https://sumo.dlr.de"
 distfiles="https://sumo.dlr.de/releases/${version}/sumo-src-${version}.tar.gz"
-checksum=dcf5339fdda0659fa44fcfd69fe3f4e03009aa9a59849624f2a3cfe72eda9f8e
+checksum=163dd6f7ed718e2a30630be3d2ac2ddfc4abce24750ed7f4efce879a3ae9447e
+
```
Comment (paper42) [inline srcpkgs/sumo/template:15]: remove this empty line

File: srcpkgs/sumo/template [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
-python_version=3
 hostmakedepends="libgdal-tools pkg-config swig python3-setuptools"
-makedepends="python3-devel ffmpeg-devel fox-devel gl2ps-devel libgdal-devel libxerces-c-devel osg-devel proj-devel"
+makedepends="python3-devel ffmpeg-devel fox-devel gl2ps-devel libgdal-devel
+ libxerces-c-devel osg-devel proj-devel"
 short_desc="SUMO (Simulation of Urban MObility) and included applications"
 maintainer="Warrior Graph <marques@larces.uece.br>"
 license="EPL-2.0"
 homepage="https://sumo.dlr.de"
 distfiles="https://sumo.dlr.de/releases/${version}/sumo-src-${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/sumo/template:13]:
```suggestion
homepage="https://sumo.dlr.de"
changelog="https://raw.githubusercontent.com/eclipse/sumo/800cb422fa4b9107a1d799d73fc19dbdd7f2cdab/docs/web/docs/ChangeLog.md"
distfiles="https://sumo.dlr.de/releases/${version}/sumo-src-${version}.tar.gz"
```

Comment (tibequadorian) [reply srcpkgs/sumo/template:13]: with that commit hash? i would use https://raw.githubusercontent.com/eclipse/sumo/master/docs/web/docs/ChangeLog.md

Comment (paper42) [reply srcpkgs/sumo/template]: I didn't notice I copied a commit hash, master will be better :)

**Pull request merged**